subprojects/base-services/src/test/groovy/org/gradle/internal/operations/DefaultBuildOperationExecutorParallelExecutionTest.groovy

```diff
@@ -44,7 +44,7 @@ class DefaultBuildOperationExecutorParallelExecutionTest extends ConcurrentSpec
     def setupBuildOperationExecutor(int maxThreads) {
         workerRegistry = new DefaultWorkerLeaseService(new DefaultResourceLockCoordinationService(), true, maxThreads)
         buildOperationExecutor = new DefaultBuildOperationExecutor(
-            operationListener, Mock(TimeProvider), Mock(ProgressLoggerFactory),
+            operationListener, Mock(TimeProvider), new NoOpProgressLoggerFactory(),
```

Comment:
@bmuschko We are not asserting any of the progress logging behavior here. A mock is no longer sufficient because all build operations are now spawning a `ProgressLogger` and calling methods on it (start, progress, complete) so we have to mock all methods. I'm not sure that verifying progress logging behavior should be a part of this or other tests. 

You might know of a better way to handle this.